QuickFlicks/Controller/MovieDetailViewController.swift

```diff
@@ -9,71 +9,141 @@
 import UIKit
 import Alamofire
 import AlamofireImage
+import SteviaLayout
 
-class MovieDetailViewController: UIViewController, UIScrollViewDelegate {
-
-    @IBOutlet weak var scrollView: UIScrollView!
+class MovieDetailView: UIView {
     
-    @IBOutlet weak var posterImageView: UIImageView!
+    var posterImageView = UIImageView()
+    var contentView = UIView()
+    var titleLabel = UILabel()
+    var dateLabel = UILabel()
+    var rateLabel = UILabel()
+    var timeLabel = UILabel()
+    var overviewLabel = UILabel()
     
-    @IBOutlet weak var titleLabel: UILabel!
+    convenience init() {
+        self.init(frame: CGRectZero)
+        
+        render()
+    }
     
-    @IBOutlet weak var dateLabel: UILabel!
+    func render() {
+        
+        backgroundColor = .whiteColor()
+        
+        // Setup UI, need to clean up later
+        // === BEGIN ===
+        posterImageView.contentMode = .ScaleAspectFill
+        
+        contentView.backgroundColor = UIColor.blackColor()
+        contentView.alpha = 0.8
+        
+        titleLabel.textColor = UIColor.whiteColor()
+        titleLabel.numberOfLines = 0
+        titleLabel.font = UIFont.boldSystemFontOfSize(17)
+        dateLabel.textColor = UIColor.whiteColor()
+        dateLabel.font = dateLabel.font.fontWithSize(13)
+        rateLabel.textColor = UIColor.whiteColor()
+        rateLabel.font = rateLabel.font.fontWithSize(13)
```

Comment:
Very minor note - in the Stevia docs, they use an example function like so:
```
    func passwordFieldStyle(f:UITextField) {
        f.secureTextEntry = true
        f.returnKeyType = .Done
    }
```

Here, I might use that for like `def labelStyle(l: UILabel)` so you don't have to type the same 13 over and over again. This makes it easier to change the font size/color later, etc. 

They call this making code "DRY". DRY= Don't Repeat Yourself